main.go

```diff
@@ -1,7 +1,328 @@
 package main
 
-import "fmt"
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"math/rand"
+	"net/http"
+	"strconv"
+	"time"
 
+	jwt "github.com/vladimiroff/jwt-go"
+
```

Comment:
Is your linter allowing this spacing, seems strange to have spaces between the github imports.